src/components/Carousel.js

```diff
@@ -330,23 +317,6 @@ class Carousel extends Component {
     }));
   };
 
-  /**
-   * Function handling touch move if drag has started. Sets dragOffset in the state.
-   * @param {event} e event
-   */
-  onTouchMove = e => {
```

Comment:
are these removed methods unneeded?

Reply:
I tested the deployed version and it seems it works as expected. Also tests e2e are passing so I guess everything works all right